getting-started/Frontend/emojify/src/main/java/com/google/cloud/kotlin/emojify/ImageActivity.kt

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.kotlin.emojify
+
+import android.graphics.Color
+import android.net.Uri
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.support.v7.widget.GridLayoutManager
+import android.support.v7.widget.RecyclerView
+import android.util.Log
+import android.util.NoSuchPropertyException
+import android.view.Menu
+import android.view.MenuItem
+import android.view.View
+import android.widget.Toast
+import com.android.volley.DefaultRetryPolicy
+import com.android.volley.Request
+import com.android.volley.Response
+import com.android.volley.toolbox.JsonObjectRequest
+import com.android.volley.toolbox.Volley
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.engine.DiskCacheStrategy
+import com.bumptech.glide.request.RequestOptions
+import com.bumptech.glide.request.target.Target
+import com.bumptech.glide.signature.ObjectKey
+import com.yanzhenjie.album.Album
+import com.yanzhenjie.album.AlbumFile
+import com.yanzhenjie.album.api.widget.Widget
+import com.yanzhenjie.album.impl.OnItemClickListener
+import com.google.firebase.storage.FirebaseStorage
+import com.google.firebase.storage.StorageMetadata
+import com.google.firebase.storage.StorageReference
+import com.yanzhenjie.album.widget.divider.Api21ItemDivider
+import kotlinx.android.synthetic.main.activity_list_content.image_view
+import kotlinx.android.synthetic.main.activity_list_content.tv_message
+import kotlinx.android.synthetic.main.activity_list_content.recycler_view
+import kotlinx.coroutines.experimental.CommonPool
+import kotlinx.coroutines.experimental.Job
+import kotlinx.coroutines.experimental.launch
+import kotlinx.android.synthetic.main.toolbar.toolbar
+import java.io.File
+import java.util.ArrayList
+import java.util.Properties
+
+class ImageActivity : AppCompatActivity() {
+
+    private lateinit var backendUrl: String
+    private var adapter: Adapter? = null
+    private var albumFiles: ArrayList<AlbumFile>? = null
```

Comment:
think you could make this a val and set it. Just if its empty to see if the user hasn't selected anything.